app/mailers/application_mailer.rb

```diff
@@ -2,6 +2,14 @@ class ApplicationMailer < Mail::Notify::Mailer
   default from: 'from@example.com'
   layout 'mailer'
 
+  def template_mail(template_id, headers)
```

Comment:
the alternative is for every mailer to make this check and all new mailers to do the same. These feels less intrusive from a developer point of view. Please let me know if there is a better way of doing this